core/modules/wdb/QueryAction.cc

```diff
@@ -300,6 +300,9 @@ class ChunkResourceRequest {
     proto::TaskMsg const& _msg;
 };
 
+util::Error makeError(sql::SqlErrorObject const& e) {
```

Comment:
Should we make sql::SqlError a subclass of the new util::Error?  Then this would (may) not be necessary?